changelog.d/15515.misc

```diff
@@ -0,0 +1 @@
+Update SQL queries to inline boolean parameters as supported in SQLite 3.27.
```

Comment:
The changelog has the wrong name:

`15515` -> `15525`

Reply:
Done.